frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/services/StartService.java

```diff
@@ -126,6 +138,23 @@ public UserStartInfo buildUserStartInfo(
                             clientRegistry.isIdentityVerificationSupported(),
                             identityEnabled);
         }
+        if (userContext.getUserProfile().filter(UserProfile::isPhoneNumberVerified).isPresent()) {
+            mfaMethodType = MFAMethodType.SMS;
+        } else if (userContext.getUserCredentials().stream()
+                .filter(t -> Objects.nonNull(t.getMfaMethods()) && !t.getMfaMethods().isEmpty())
+                .anyMatch(
```

Comment:
We've got two nested `anyMatch` calls here, can we simplify this condition?

Reply:
That's much nicer!